### PR TITLE
Fixed time zone related test cases by adding UTC time zone condition

### PR DESCRIPTION
**Pull request by v-baambati**
1. Added/Updated time zone related test cases.
2. Added env condition check for all the time zone related test cases.
3. Enables time zone related test cases for all the time zones along with env check.

### CODE REVIEW

File: packages/react-charting/UnitTests/AreaChartUT.test.tsx [hunk 1]
```diff
@@ -637,8 +643,11 @@ runTest('_getAriaLabel', () => {
     container => {
       const points = getById(container, /circle/i);
       expect(points).toHaveLength(1);
-      expect(points[0].getAttribute('aria-label')).toEqual('Jan 1, 2022. Legend 1, 10.');
+      expect(points[0].getAttribute('aria-label')).toMatchSnapshot();
     },
+    undefined,
+    undefined,
+    !isTimezoneSet(Timezone.UTC),
```
Comment (AtishayMsft) [inline packages/react-charting/UnitTests/AreaChartUT.test.tsx:650]: what are you doing here.

Comment (v-baambati) [reply packages/react-charting/UnitTests/AreaChartUT.test.tsx:650]: We are planning to execute these test cases only in UTC time zone and we are passing UTC time sone as input to this function.

**Comment by fabricteam** [comment]
## 🕵 fluentuiv8 No visual regressions between this PR and main
<div id="vrtCommentfluent-ui_VRT_Pipeline_v8"/>

File: packages/react-charting/UnitTests/HeatMapChartUT.test.tsx [hunk 2]
```diff
@@ -574,19 +576,22 @@ runTest('_createNewDataSet', () => {
 });
 
 runTest('Skip - timezone related test cases', () => {
```
Comment (AtishayMsft) [inline packages/react-charting/UnitTests/HeatMapChartUT.test.tsx:578]: remove the skip tag. Here and everywhere else

File: packages/react-charting/UnitTests/HeatMapChartUT.test.tsx [hunk 3]
```diff
@@ -574,19 +576,22 @@ runTest('_createNewDataSet', () => {
...
-    });
-    expect(instance).toBeDefined();
-    const result = instance._createNewDataSet(HeatMapDataDatePoints, '%b/%d', '.2~s', '%b/%d', '.2~s');
-    expect(result.xAxisPoints[0]).toEqual('Fri Apr 03 2020 05:30:00 GMT+0530 (India Standard Time)');
-    expect(result.yAxisPoints[0]).toEqual('Tue Mar 03 2020 05:30:00 GMT+0530 (India Standard Time)');
-  });
-
-  test('Should return proper aria-label for date xPoint and yPoint', () => {
+  conditionalTest(isTimezoneSet(Timezone.UTC))(
+    'Should return proper data set for date axis with default axis type',
```
Comment (AtishayMsft) [inline packages/react-charting/UnitTests/HeatMapChartUT.test.tsx:580]: What about date object based tests in LinChart, Vertical Bar Chart and Vertical Stacked Bar Chart.

File: packages/react-charting/src/components/VerticalStackedBarChart/VerticalStackedBarChartRTL.test.tsx
Comment (krkshitij) [inline packages/react-charting/src/components/VerticalStackedBarChart/VerticalStackedBarChartRTL.test.tsx:1]: Apply the TEST env check in the following line:
https://github.com/microsoft/fluentui/blob/34f1202a2045955c1c225d5bb8e76a88d8c614df/packages/react-charting/src/components/VerticalStackedBarChart/VerticalStackedBarChartRTL.test.tsx#L185

File: packages/react-charting/src/components/VerticalStackedBarChart/VerticalStackedBarChartRTL.test.tsx [hunk 4]
```diff
@@ -133,7 +134,7 @@ describe('Vertical stacked bar chart rendering', () => {
     },
     undefined,
     undefined,
-    !isTimezoneSet(Timezone.UTC),
+    !(isTimezoneSet(Timezone.UTC) && env === 'TEST'),
```
Comment (AtishayMsft) [inline packages/react-charting/src/components/VerticalStackedBarChart/VerticalStackedBarChartRTL.test.tsx:137]: foreach timezone loop needs to be applied for vertical bar chart, vertical stacked bar chart and line chart.

**Review by AtishayMsft — APPROVED** [review]

File: packages/react-charting/src/components/VerticalStackedBarChart/VerticalStackedBarChartRTL.test.tsx (hunk 4 above)
Comment (AtishayMsft) [reply packages/react-charting/src/components/VerticalStackedBarChart/VerticalStackedBarChartRTL.test.tsx:137]: Needs to be fixed in the next iteration.

File: packages/react-charting/UnitTests/HeatMapChartUT.test.tsx (hunk 3 above)
Comment (AtishayMsft) [reply packages/react-charting/UnitTests/HeatMapChartUT.test.tsx]: Needs to be fixed in the next iteration.

**Pull request merged**